client/state/action-types.js

```diff
@@ -328,6 +328,7 @@ export const SEO_TITLE_SET = 'SEO_TITLE_SET';
 export const SIGNUP_DEPENDENCY_STORE_RESET = 'SIGNUP_DEPENDENCY_STORE_RESET';
 export const SIGNUP_DEPENDENCY_STORE_UPDATE = 'SIGNUP_DEPENDENCY_STORE_UPDATE';
 export const SIGNUP_STEPS_SITE_TITLE = 'SIGNUP_STEPS_SITE_TITLE';
+export const SIGNUP_STEPS_SURVEY = 'SIGNUP_STEPS_SURVEY';
```

Comment:
Shouldn't we add a verb to the end of the action name? As recommended by [the docs on actions](https://wpcalypso.wordpress.com/devdocs/docs/our-approach-to-data.md#actions)? Maybe `SIGNUP_STEPS_SURVEY_SUBMIT` or `SIGNUP_STEPS_SURVEY_SET`.

That also goes for `SIGNUP_STEPS_SITE_TITLE`.